tests/main/auto-refresh-pre-download/task.yaml

```diff
@@ -114,6 +117,7 @@ execute: |
 
     if retry -n 5 sh -c 'snap changes | tail -n 2 | grep "Done.*Auto-refresh.*snap.*test-snapd-sh"'; then
       echo "unexpected auto-refresh of test-snapd-sh"
+      tests.cleanup restore
```

Comment:
this is not needed, no following waits but maybe keeping it for consistency would be nice?

Reply:
I agree, assuming the process remains the same this one will do nothing, but I wanted to be extra safe and it didn't cost much